src/main/kotlin/net/maizegenetics/phgv2/agc/AnnotateFastas.kt

```diff
@@ -15,28 +15,32 @@ import org.apache.logging.log4j.LogManager
 import java.io.File
 
 /**
- * This class takes a list of fasta files and updates each idline with the sample name.
- * The sample name is derived from the fasta file name, minus the extension.
- * Fasta files should be named with just the sample name, extension can be either .fa or .fasta.
- * These updates are required as AGC processing maintains  fasta comments, but it does not maintain the
- * sample name when included with a query request.
+ * This class takes a keyfile containing a list of fasta files and their sample names.
```

Comment:
Consider updating example usage in README

Reply:
added to README.md